src/Orc.Controls/Orc.Controls.Shared/Controls/ColorPicker/Themes/ColorPicker.generic.xaml

```diff
@@ -455,8 +455,13 @@
                                     </TabItem>
                                 </TabControl>
 
-                                <Grid Grid.Row="1" Grid.ColumnSpan="2" Height="20" Margin="2 0 2 2" HorizontalAlignment="Stretch" VerticalAlignment="Center">
-                                    <ComboBox x:Name="ComboBoxColor" Height="20" VerticalAlignment="Center" HorizontalAlignment="Stretch" Padding="6 3"/>
+                                <Grid Grid.Row="1" Grid.ColumnSpan="2" Height="25" Margin="2 0 2 2" HorizontalAlignment="Stretch" VerticalAlignment="Center">
```

Comment:
Why set this height manually? Is it really required? It'd like to have as less customized xaml code as possible.